clients/client/src/main/java/org/projectnessie/client/api/GetAllReferencesBuilder.java

```diff
@@ -36,10 +42,15 @@
    */
   GetAllReferencesBuilder fetch(FetchOption fetchOption);
 
-  /**
-   * Fetches all references and returns them in a {@link ReferencesResponse} instance.
-   *
-   * @return Fetches all references and returns them in a {@link ReferencesResponse} instance.
-   */
+  @Override
   ReferencesResponse get();
```

Comment:
Do we want to deprecate this method too? The `@Deprecated` annotation on the overridden method does not normally cause warnings on the more specific method.

By the way, it has usages not converted to `stream()` in `AbstractRestReferences`... Was that intentional?

Reply:
I've removed the deprecation, because as you mentioned, people may want to use explicit paging.

Reply:
+1 to removing deprecation, but do we need this override now?

Reply:
nope